src/pages/styles/icons.tsx

```diff
@@ -92,6 +104,12 @@ const Page = (): JSX.Element => (
         exampleFirst={false}
         noCode
       >
+        <ExternalLink
```

Comment:
I'd maybe move this after the icon list. Somehow would feel more intuitive there. So the user would first see the icons and then have an option to go see some more detailed info if they wish.

Reply:
I tried it on the end. To me that doesn't look so "aesthetic" and the link is a bit confusing when it's close to the next heading. Anyways, I don't have a strong opinion on this.

Reply:
I personally like it better on the bottom of the section. A little inline styling, like `style={{ display: 'inline-block', marginBottom: '20px' }}` would likely solve the issue of being too close to the following heading.

Reply:
A third opinion would likely be a good idea though :)

Reply:
As discussed elsewhere, let's leave it as it is, since neither option is clearly superior to the other.